wallets/server.js

```diff
@@ -82,19 +82,18 @@ export async function createInvoice (userId, { msats, description, descriptionHa
 export async function createWrappedInvoice (userId,
   { msats, feePercent, description, descriptionHash, expiry = 360 },
   { predecessorId, models, me, lnd }) {
-  let logger, bolt11
-  try {
-    const { invoice, wallet } = await createInvoice(userId, {
-      // this is the amount the stacker will receive, the other (feePercent)% is our fee
-      msats: toPositiveBigInt(msats) * (100n - feePercent) / 100n,
-      description,
-      descriptionHash,
-      expiry
-    }, { predecessorId, models })
+  const { invoice, wallet } = await createInvoice(userId, {
+    // this is the amount the stacker will receive, the other (feePercent)% is our fee
+    msats: toPositiveBigInt(msats) * (100n - feePercent) / 100n,
+    description,
+    descriptionHash,
+    expiry
+  }, { predecessorId, models })
 
-    logger = walletLogger({ wallet, models })
-    bolt11 = invoice
+  const logger = walletLogger({ wallet, models })
+  const bolt11 = invoice
```

Comment:
if the only thing this catch does is logging the error and rethrow, there is no point in wrapping all the rest of the code in the try-catch, also eslint didn't like this.